src/lib/skilling/skills/herblore/mixables/barbMixes.ts

```diff
@@ -0,0 +1,390 @@
+import { Bank } from 'oldschooljs';
+
+import itemID from '../../../../util/itemID';
+import { Mixable } from '../../../types';
+
+const barbMixes: Mixable[] = [
+	{
+		name: 'Attack mix(2)',
+		aliases: ['attack mix roe', 'attack mix(2)', 'attack mix 2 roe'],
+		id: itemID('Attack mix(2)'),
```

Comment:
instead of having name and id, you can have just `item: getOSItem()` and now you have the full item (id and name included), AND you dont have to worry about typos/mismatches on the name or id.